src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/entity/CsipRecord.kt

```diff
@@ -57,7 +65,7 @@ data class CsipRecord(
 
   @Column(length = 255)
   val lastModifiedByDisplayName: String? = null,
-) : AbstractAggregateRoot<CsipRecord>() {
+) : Serializable, AbstractAggregateRoot<CsipRecord>() {
```

Comment:
More for my interest, what's the purpose of Serializable here?

Reply:
It might not be needed any more...if I could remember exactly the reason why it _was_ needed I'll send you a message!